src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/trust/TrustDataDto.java

```diff
@@ -0,0 +1,112 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.dto.trust;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class TrustDataDto {
+
+    @JsonProperty("trust_id")
+    private String trustId;
+
+    @JsonProperty("trust_name")
+    private String trustName;
+
+    @JsonProperty("trust_creation_date_day")
+    private String trustCreationDateDay;
+
+    @JsonProperty("trust_creation_date_month")
+    private String trustCreationDateMonth;
+
+    @JsonProperty("trust_creation_date_year")
+    private String trustCreationDateYear;
+
+    @JsonProperty("unable_to_obtain_all_trust_info")
+    private Boolean unableToObtainAllTrustInfo;
+
+    @JsonProperty("INDIVIDUAL")
+    private List<IndividualDto> individualBeneficialOwners;
+
+    @JsonProperty("HISTORICAL_BO")
+    private List<HistoricalBoDto> historicalBeneficialOwners;
+
+    @JsonProperty("CORPORATE")
+    private List<CorporateDto> corporateBeneficialOwners;
+
+    public String getTrustId() {
+        return trustId;
+    }
+
+    public void setTrustId(String trustId) {
+        this.trustId = trustId;
+    }
+
+    public String getTrustName() {
+        return trustName;
+    }
+
+    public void setTrustName(String trustName) {
+        this.trustName = trustName;
+    }
+
+    public String getTrustCreationDateDay() {
+        return trustCreationDateDay;
+    }
+
+    public void setTrustCreationDateDay(String trustCreationDateDay) {
```

Comment:
Wondering why in this DTO the date component fields are returned individually and not as a single date (string)?